src/index.ts

```diff
@@ -24,16 +24,17 @@ export interface Options {
 }
 
 /**
- * Returns the base URL while taking into account the GCE_METADATA_IP
```

Comment:
Could we add a blurb in the README that explains this thing?  Otherwise I don't know how folks would know it exists.

Reply:
So, I added a pretty light description in the README, following the style of the rest of the doc. I can go more in-depth though if it's preferred.